image-loader/app/lib/imaging/MimeTypeDetection.scala

```diff
@@ -27,6 +30,14 @@ object MimeTypeDetection extends GridLogging {
     case Failure(_: Throwable) => Left(new UnsupportedMimeTypeException(FALLBACK))
   }
 
+  // DNG files look like Tiff files, but include a DNGVersion tag (0xC612)
+  // See https://www.adobe.com/content/dam/acom/en/products/photoshop/pdfs/dng_spec_1.4.0.0.pdf
+  private def isDng(file: File) = {
+    val metadata = TiffMetadataReader.readMetadata(file)
+    val directory = metadata.getFirstDirectoryOfType(classOf[ExifIFD0Directory])
+    directory.containsTag(0xC612)
```

Comment:
nit: I think technically this could NPE (if somehow there wasn't a 0th ifd on the tiff), other places in the code we've  checked that it's not null